code/src/App.js

```diff
@@ -1,12 +1,34 @@
 import React from 'react';
 import data from './data.json';
+import Header from './components/Header'
+import Cover from './components/Cover'
+import Albumname from './components/Albumname';
+import Artistname from './components/Artistname';
+import "./index.css";
 
-console.log(data);
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
+    <div className='all-container'>
+      <div className="Header"> 
+       <Header /> 
+      </div>
+      <div className='music-container'>
+      {data.albums.items.map (item => {
+       return <>
+        <div className='theAlbums'>
+          <Cover key={item.id} data={item} img={item.images}/> 
+          <Albumname Albumname={item.name} urlAlbumname={item.name}/>
+          <Artistname artists={item.artists} artistUrl={item.artists}/>
+        </div>
+        </>
+      })}
+      </div>
+      <div className='Footer'>
+     <h3>Project Music Release for Technigo by Viktor & Kartika</h3>
+     </div>
     </div>
   );
 }
+
+//we're declaring a variable "data" and {item} is the value. so we connect the varibale Line 15
```

Comment:
If you want to add comments in JSX you have to wrap the comment inside {}.
Adding the comment at the end is a bit problematic since the line number might change by the time you are finished with the project, or if you come back to the project to make changes you will have to keep in mind to also change your comments. :)